src/J2N.Randomizer/J2N.Randomizer.csproj

```diff
@@ -0,0 +1,56 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net9.0;net8.0;net6.0;netstandard2.1</TargetFrameworks>
```

Comment:
If this project is only included for netstandard2.1, are the other target frameworks needed here? I don't see a reason to reject this PR over this, but just curious.

Reply:
Well, at minimum we will need `net6.0` to ensure the Random overloads exist for consumers in .NET Core. Without that, the fix doesn't work. It is also important to override any additional virtual methods that Microsoft adds in the future.

Granted, `net8.0` and `net9.0` will have a type conflict if both packages are installed, but it is not very likely to happen and we explicitly state this package isn't intended for direct consumption.